taqueria-plugin-tezos-client/common.ts

```diff
@@ -2,10 +2,15 @@ import { getArchSync, getDockerImage } from '@taqueria/node-sdk';
 import { RequestArgs } from '@taqueria/node-sdk/types';
 import { join } from 'path';
 
-export const getFlextesaImage = (arch: 'linux/arm64/v8' | 'linux/amd64'): string =>
+// Should point to the latest stable version, so it needs to be updated as part of our release process.
+const getFlextesaImage = (arch: 'linux/arm64/v8' | 'linux/amd64'): string =>
 	arch === 'linux/arm64/v8' ? 'oxheadalpha/flextesa:20221026' : 'oxheadalpha/flextesa:20221026';
```

Comment:
Are these the same image? If so, why not define in a constant?

Also, is a ternary operator even needed here?

Reply:
Sometimes they might be different because of how that particular team handles the image release

Reply:
So this would be used if there were separate images for each architecture?

Reply:
Yep!